doc/source/reference/routines.linalg.rst

```diff
@@ -18,6 +18,25 @@ or specify the processor architecture.
 .. _OpenBLAS: https://www.openblas.net/
 .. _threadpoolctl: https://github.com/joblib/threadpoolctl
 
+Frequently asked question about `numpy.linalg` include "Why do both NumPy and
+SciPy have `linalg` submodules, with duplicated functions?" and "Which should
+I use?"  The following is from the SciPy FAQ:
+
+* `scipy.linalg` is a more complete wrapping of Fortran LAPACK using f2py.
+* One of the design goals of NumPy was to make it buildable without a Fortran
+  compiler, and if you don’t have LAPACK available, NumPy will use its own
+  implementation. SciPy requires a Fortran compiler to be built, and heavily
+  depends on wrapped Fortran code.
```

Comment:
although this is taken from the SciPy FAQ, I don't know if this is so relevant now since most users will not be building their own NumPy or SciPY

Reply:
Yeah, I like the other doc string. I am not sure I feel we need this long copy/repeat of the SciPy FAQ. Is there a short suggestion aside from SciPy being more complete, e.g. to prefer SciPy unless you want to avoid the dependency?

Reply:
> prefer SciPy unless you want to avoid the dependency?

I think that's the key message. Then a short historical note like "this duplication stems from a time when NumPy and SciPy were quite hard to install" would probably be appropriate

Reply:
@seberg wrote:

> Yeah, I like the other doc string. 

What other docstring are you referring to?

> Is there a short suggestion aside from SciPy being more complete, e.g. to prefer SciPy unless you want to avoid the dependency?

I agree that a shorter version would be better, and quotes directly from the SciPy FAQ can be dropped.

Reply:
Sorry, I meant the `See Also` changes being great.

Reply:
OK, got it.

Reply:
I pushed a shorter version.  I've refrained from giving a recommendation for which to use, and I didn't include the short historical note that Ralf suggested.  So now it is just an acknowledgment of the existence of `scipy.linalg`, and a few brief comments about how it is different from `numpy.linalg`.